WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -163,6 +167,28 @@ extension EnvironmentValues {
     }
 }
 
+@available(iOS 17.0, *)
+private extension PointOfSaleDashboardView {
+    func enableCoupons() async {
```

Comment:
One more comment regarding that: when the button is tapped, we should make sure the item state reflects that and the view doesn't keep being stale. So it should switch to a loading state.